### PR TITLE
This is bad and I still have no clue how I got it to even build

### PR DESCRIPTION
**Pull request by sillypears**
/shrug emoji thing

### CODE REVIEW

File: MacSG/frmMain.vb [hunk 1]
```diff
@@ -235,9 +235,10 @@ Public Class frmMain
 
 
     'Unattached subs
-    Public Sub genStream(streamer As String, quality As String, source As String, windowTitle As String, configFile As String)
-
-        Dim strLivestreamerProcess As New ProcessStartInfo("cmd.exe", "/c title " & windowTitle & " & " & source & streamer & quality & "--player-args "" --config %AppData%\MacSG\" & configFile & " {filename}")
+    Public Sub genStream(streamer As String, quality As String, source As String, windowTitle As String, configFile As String, racerNumber As String)
+        Dim runningProcess = "/c title " & windowTitle & " & " & source & "-a "" --config %AppData%\MacSG\vlcrc --width 877 --height 518 -"" " & " --title " & racerNumber & " --hls-live-edge 1 twitch.tv/" & streamer & quality
+        Dim strLivestreamerProcess As New ProcessStartInfo("cmd.exe", runningProcess)
+        'MessageBox.Show(runningProcess)
```
Comment (alexismartin) [inline MacSG/frmMain.vb]:
```suggestion
```
Remove leftover comment

**Review by alexismartin — CHANGES_REQUESTED** [review]

File: MacSG/frmMain.vb (hunk 1 above)
Comment (sillypears) [reply MacSG/frmMain.vb]: Oke

729d064

**Review by alexismartin — APPROVED** [review]

**Comment by sillypears** [comment]
It works on my machine after I build it.

**Comment by alexismartin** [comment]
Can you figure out how to update the installer in https://github.com/necrommunity/macsg/tree/master/Setup/Release and maybe create a new release? Not sure how @MacND did it.

**Pull request merged**